neuron_morphology/transforms/tilt_correction/compute_tilt_correction.py

```diff
@@ -0,0 +1,226 @@
+from typing import Dict, Optional, List, Any
+
+import logging
+import copy as cp
+
+import numpy as np
+from scipy.spatial.distance import euclidean
+import pandas as pd
+import h5py
+
+from argschema.argschema_parser import ArgSchemaParser
+from neuron_morphology.transforms.tilt_correction._schemas import (
+    InputParameters, OutputParameters)
+from neuron_morphology.morphology import Morphology
+from neuron_morphology.swc_io import morphology_from_swc
+import neuron_morphology.transforms.affine_transform as aff
+
+
+CCF_SHAPE = (1320, 800, 1140)
+CCF_RESOLUTION = 10
+
+
+def get_tilt_correction(morphology: Morphology,
+                        soma_voxel: Dict[str, int],
+                        slice_angle_matrix: float,
+                        closest_path,
+                        ):
+    """
+        Find the tilt angle between the slice plane and the nearest streamline
+
+        Parameters
+        ----------
+        morphology: Morphology object
+        soma_voxel: soma voxel in ccf {'x': , 'y': , 'z': }
+        slice_angle_matrix: 4 x 4 affine matrix of the slice plane
+            relative to ccf
+        closest_path: 3 x N array of voxel coordinates in closest streamline
```

Comment:
It looks below like you just use the first (pia) and last (wm) coordinates in this array. Why pass the whole thing rather than explicitly passing pia and wm side path coordinates?

Reply:
I left it as is partially because of your next comment - it seems reasonable that in future the tilt correction is changed to local streamline angle based on the curvature of the streamline.

Reply:
could you document this? Passing around data that we don't use now but might use in the future seems sketchy